src/components/Radio/Radio.tsx

```diff
@@ -79,5 +102,5 @@ export const Radio = ({ label, checked, onChange }: RadioProps) => (
 );
 
 Radio.defaultProps = {
-  checked: false,
+  // checked: false,
 };
```

Comment:
I don't think the `defaultProps` are helping anything here. What do you think about removing them?